it_hardware/data/product_template.xml

```diff
@@ -0,0 +1,94 @@
+        <field name="image_1920" type="base64" file="it_hardware/static/src/binary/product_template/10-image_1920"/>
+    </record>
+    <record id="product_template_11" model="product.template" context="{'create_product_product': False}">
+        <field name="name">CCTV Camera</field>
+        <field name="detailed_type">product</field>
+        <field name="categ_id" ref="product.product_category_all"/>
+        <field name="list_price">52.01</field>
+        <field name="use_expiration_date" eval="True"/>
+        <field name="tracking">serial</field>
+        <field name="x_warranty_days">100</field>
+        <field name="image_1920" type="base64" file="it_hardware/static/src/binary/product_template/11-image_1920"/>
+    </record>
+    <record id="product_template_6" model="product.template" context="{'create_product_product': False}">
+        <field name="name">Warranty Service</field>
+        <field name="detailed_type">service</field>
+        <field name="categ_id" ref="product.product_category_all"/>
+        <field name="project_id" ref="industry_fsm.fsm_project"/>
+        <field name="service_tracking">task_global_project</field>
+        <field name="image_1920" type="base64" file="it_hardware/static/src/binary/product_template/6-image_1920"/>
+    </record>
+    <record id="product_template_7" model="product.template" context="{'create_product_product': False}">
+        <field name="name">ThinkPad X1 Yoga 35.56cms - 12th Gen Intel i7</field>
+        <field name="detailed_type">product</field>
+        <field name="categ_id" ref="product.product_category_all"/>
+        <field name="list_price">2813.29</field>
+        <field name="x_warranty_days">465</field>
+        <field name="use_expiration_date" eval="True"/>
+        <field name="tracking">lot</field>
+        <field name="optional_product_ids" eval="[(6, 0, [ref('product_template_6')])]"/>
+        <field name="description_sale">A 35.5cms (14) 2-in-1 touchscreen laptop, the ThinkPad X1 Yoga Gen 7 delivers features that matter.
+        Enjoy the spectacular Communications Bar that raises the bar on collaboration.
+        Robust security includes Tile&#174; tracking technology&#8212;just in case.
+        Plus, high-performance 12th Gen Intel&#174; Core&#8482; vPro processing, on the Intel&#174; Evo&#8482; platform provides super-responsive, on-the-go power.</field>
+        <field name="image_1920" type="base64" file="it_hardware/static/src/binary/product_template/7-image_1920"/>
+    </record>
+    <record id="product_template_5" model="product.template" context="{'create_product_product': False}">
+        <field name="name">ThinkPad X1 Carbon 35.56cms - 12th Gen Intel i7</field>
+        <field name="detailed_type">product</field>
+        <field name="categ_id" ref="product.product_category_all"/>
+        <field name="list_price">3780.49</field>
+        <field name="expiration_time">365</field>
+        <field name="x_warranty_days">0</field>
```

Comment:
Before removing all expiration_time, make sure that you update correctly all x_warranty_days to the same value
```suggestion
        <field name="x_warranty_days">365</field>
```